tests/ethereum/main.rs

```diff
@@ -192,23 +197,14 @@ fn run_test_unit(path: &Path, unit: &TestUnit) {
                     // This is a good reference for a minimal state/DB commitment logic for
                     // PEVM/REVM to meet the Ethereum specs throughout the eras.
                     for (address, account) in state {
-                        if !account.is_touched() {
-                            continue;
-                        }
-                        if account.is_selfdestructed()
-                            || (account.is_empty()
-                                && spec_id.is_enabled_in(SpecId::SPURIOUS_DRAGON))
-                        {
+                        if let Some(account) = account {
```

Comment:
Big win for library users. Hopefully, we don't miss any information for nodes to sync correctly.